assets/js/components/indicators.js

```diff
@@ -298,12 +298,18 @@ export class IndicatorSection {
     this.comparisonButtonsContainer.insertBefore(this.chartContainer);
   }
 
+  highlightComparisonButton(section, muniId) {
+    $(section + ' .chart-btn').css('border', 'none');
```

Comment:
@michaelglenister lets use semantic classes even from JavaScript rather than targeting direct CSS attributes e.g.
```
$(section + ' .chart-btn'). classList.add('active')
...
.active {
  border: none
}
```

Reply:
Changed, I used addClass() from jQuery instead of classList.add()